### PR TITLE
add notes for n26158

### PR DESCRIPTION
**Pull request by stickies-v**
cc @furszy

### CODE REVIEW

File: _posts/2022-10-12-#26158.md [hunk 1]
```diff
@@ -7,17 +7,44 @@ authors: [furszy]
...
+
+1. Are there any downsides to `make check` only running `-priority-level=high` benchmarks by default? Do you think the benefits outweigh the downsides?
+
+1. Why do we need to `static_cast<PriorityLevel>` the return value when [`operator|(PriorityLevel a, PriorityLevel b)`](https://github.com/bitcoin-core-review-club/bitcoin/blob/cbc077e77a5d0ba8ae11b1f5d89480c997fdef55/src/bench/bench.h#L50)'s return type is `PriorityLevel` already?
+
+1. In which sense is `PriorityLevel` an enum? In which sense is it a bit vector? What do you think about this approach?
+
+1. Which levels can we pass to `-priority-level`? Are they all represented in [`map_priority_level`](https://github.com/bitcoin-core-review-club/bitcoin/blob/cbc077e77a5d0ba8ae11b1f5d89480c997fdef55/src/bench/bench.cpp#L47-L51)? How are the other one(s) resolved? Can you think of a different approach?
+
+1. Were you able to compile and run the benchmarks? How did you, or could you, test that the `-priority-level` argument works as expected?
```
Comment (glozow) [inline _posts/2022-10-12-#26158.md:36]:
```suggestion
1. Were you able to compile and run the benchmarks? How did you test that the `-priority-level` argument works as expected?
```

File: _posts/2022-10-12-#26158.md [hunk 2]
```diff
@@ -7,17 +7,44 @@ authors: [furszy]
...
+
+1. Which benchmarks do you think should/could be labeled as `LOW` or `MEDIUM` priority? What did you base that decision on?
+
+1.  Which of the below do you think are valid `PriorityLevel`s:
```
Comment (glozow) [inline _posts/2022-10-12-#26158.md:40]:
```suggestion
1.  Quiz: Which of the following are valid `PriorityLevel`s:
```

**Review by glozow — COMMENTED** [review]
Thanks @stickies-v! Looks good to me, just nits.

**Comment by stickies-v** [comment]
Thanks glozow, incorporated all your suggestions!

**Pull request merged**